### PR TITLE
Improve testing

### PR DESCRIPTION
**Pull request by Jamim**
Hello,

I believe it would be nice to have tests for the upcoming Python versions.

These changes also:
 - add more precise language classifiers at `setup.py`
 - remove `tox-venv` since it causes failure on CI for Python 3.5
 - remove [deprecated](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration) `sudo` option from `.travis.yml`

Regards!

### CODE REVIEW

**Comment by Jamim** [comment]
Hello @converge,
Can you please review/merge this PR?
Thank you!

File: tox.ini [hunk 1]
```diff
@@ -26,5 +25,5 @@ skip_install = True
 [testenv:black]
 basepython = python3.7
 deps = black
-commands = black -t py33 --check .
+commands = black -t py37 --check .
```
Comment (breuerfelix) [inline tox.ini]: this needs to be 33 because we have to target 33 since we support versions which are lower than 3.7 :)

Comment (Jamim) [reply tox.ini]: Hello @breuerfelix,

As I see at `setup.py`, InstaPy [doesn't actually support](https://github.com/timgrossmann/InstaPy/blob/master/setup.py#L88) Python 3.3.
```python
    python_requires=">=3, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
```

Should I change the target to `py34`?

Comment (breuerfelix) [reply tox.ini]: actually thats true... I don't know why we don't have 3.3 :D but yeah change it to 3.4 :) that would be cool !

**Comment by breuerfelix** [comment]
really cool changes ! I didnt know that this is possible :D always happy to learn new stuff :)

File: tox.ini (hunk 1 above)
Comment (Jamim) [reply tox.ini]: It'd done.

**Pull request merged**